com.unity.multiplayer.mlapi/Tests/Runtime/NetworkTransform/NetworkTransformStateTests.cs

```diff
@@ -0,0 +1,183 @@
+using NUnit.Framework;
+using Unity.Netcode.Prototyping;
+using UnityEngine;
+
+namespace Unity.Netcode.RuntimeTests
+{
+    public class NetworkTransformStateTests
```

Comment:
I'd be interested to see a multiinstance test where we check that values that aren't supposed to change actually didn't change client side. ex: server side shouldn't pos.x, updates pos.x and pos.y, make sure only receive pos.y and pos.x stays the same (could have initial value != 0 to make sure it doesn't get reset to 0). I'm sure it works with your current implementation, this test would be for regression testing, making sure that future work doesn't break this.